### ISSUE TITLE
Repair not clearing

### ISSUE THREAD

**Issue by erkr** [frenck/spook#753]
### What version of Spook are you using?

3.0.1

### What version of Home Assistant are you using?

2024.5.2

### The problem

Thanks for the usefully checks provided!
During startup I always have 2 ‘temporary’ repairs for two sensors from the Google Drive Backup add-on. That is by itself correct as that add on mystically adds these sensors with some delay without providing an integration owning those sensors.

Now the bug.:
Once the backup sensors appear, normally the repairs get automatically cleared, but not always!
Example today for the ‘sensor.backup_state’. 40 minutes after the sensor was added, the repair is still there:
![image](https://github.com/frenck/spook/assets/4363599/927660c0-5534-4fa6-9e1a-d6b7c3b22435)

For some reason that is not always noticed by Spook. Of course it is easily ignored, but that shouldn’t be needed

**Comment by frenck** [comment]
> During startup I always have 2 ‘temporary’ repairs for two sensors from the Google Drive Backup add-on.

This add-on is bypassing all Home Assistant logic and writing directly into the state machine (which it shouldn't do). It should use an integration. Please report it with the add-on author. 

This is not an issue that we can ever solve. This construct the add-on is using can cause more issues with Home Assistant.

../Frenck

**Comment by erkr** [comment]
add on issue already exists: https://github.com/sabeechen/hassio-google-drive-backup/issues/805

**Comment by frenck** [comment]
> I was only referring that the repair doesn't clear over time.

Correct, as this is not handled in the correct way by the add-on. It injects directly into the state machine, bypassing all logic in Home Assistant.

**Comment by erkr** [comment]
Thanks, will apply the rest workaround mentioned in the issue
